packages/next/server/config-shared.ts

```diff
@@ -128,6 +128,10 @@ export interface ExperimentalConfig {
       skipDefaultConversion?: boolean
     }
   >
+  swcTraceProfiling?: {
```

Comment:
Let's make this `swcTraceProfiling: boolean` instead of having extra config options the default for it is fine.